### PR TITLE
Add script covering case when UI.PerformInteraction gets UNSUPPORTED_…

### PR DESCRIPTION
**Pull request by vvvakulenko**
…RESOURCE and VR.PerformInteraction gets WARNINGS from HMI
Script is a part of TS #666

### CODE REVIEW

**Comment by vvvakulenko** [comment]
@istoimenova , Please review

File: test_scripts/API/PerformInteraction/025_ATF_UI_IsReady_no_resp_PerfInteract_UI_PerfInterac_UNSUP_RES.lua [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        EXPECT_HMICALL("BasicCommunication.ActivateApp")
+        :Do(function(_,data1)
+          self.hmiConnection:SendResponse(data1.id,"BasicCommunication.ActivateApp", "SUCCESS", {})
+        end)
+      end)
+    end
+  end)
+  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+end
+
```
Comment (istoimenova) [inline test_scripts/API/PerformInteraction/025_ATF_UI_IsReady_no_resp_PerfInteract_UI_PerfInterac_UNSUP_RES.lua]: @vvvakulenko, as precondition for CreateIntercationChoiceSet, you should have icon.png on DUT storage. You can use as example:
commonSteps:PutFile("Precondition_PutFile", "icon.png")

Comment (vvvakulenko) [inline test_scripts/API/PerformInteraction/025_ATF_UI_IsReady_no_resp_PerfInteract_UI_PerfInterac_UNSUP_RES.lua]: @istoimenova , updated in commit 1e3261539463f462759285b51b796092a74abae8

**Comment by vvvakulenko** [comment]
@istoimenova , Please check updated

**Pull request merged**